tree-view-sample-plugin/src/comments.ts

```diff
@@ -0,0 +1,170 @@
+import * as theia from '@theia/plugin';
+
+const comments: Map<string, string[]> = new Map<string, string[]>();
+
+comments.set('Ann', ['Great plugin!', 'Can you share the link to this example?']);
+comments.set('Yevhen', ['Cleanup your code please']);
+comments.set('Sun', ['Do not forget about docs']);
+
+const ON_DID_SELECT_USER = 'comments-tree-on-did-select-user';
+const ON_DID_SELECT_COMMENT = 'comments-tree-on-did-select-comment';
+
+export class Comments {
+
+    treeDataProvider: TestDataProvider;
+    tree: theia.TreeView<string>;
+
+    selectedUser: string | undefined = undefined;
```

Comment:
Up to you, but by default `selectedUser` is already undefined: [Playground sample](https://www.typescriptlang.org/play/#src=%0D%0Aclass%20Comments%20%7B%0D%0A%20%20%20%20selectedUser%3A%20string%20%7C%20undefined%0D%0A%7D%0D%0A%0D%0Aconsole.log(new%20Comments().selectedUser)%0D%0A%0D%0Aconst%20resultSpan%20%3D%20document.createElement(%22span%22)%0D%0AresultSpan.innerText%20%3D%20%22Selected%20user%20value%20%3D%20%22%20%2B%20new%20Comments().selectedUser%0D%0Adocument.body.append(resultSpan))

Reply:
Yes, indeed. Will be fixed. Thanks.